docs/technical-documentation/install-enable-drupal-modules.md

```diff
@@ -13,7 +13,6 @@ $ composer require "islandora/islandora_defaults:1.0.0"
 In the [Islandora 8 playbook](https://github.com/Islandora-Devops/claw-playbook), you can add a Drupal module's or theme's machine name to the `drupal_composer_dependencies` variable [here](https://github.com/Islandora-Devops/islandora-playbook/blob/master/inventory/vagrant/group_vars/webserver/drupal.yml). 
 
```

Comment:
Oops.  Sorry if that wasn't clear.  I meant this line, to collapse the paragraphs that talk about ansible variables.

Reply:
Still not sure what you're looking for. Line #?